libcxx/include/__functional/operations.h

```diff
@@ -353,11 +352,11 @@ struct _LIBCPP_TEMPLATE_VIS equal_to<void>
 #endif
 
 template <class _Tp>
-struct __is_trivial_equality_predicate<equal_to<_Tp>, _Tp, _Tp> : true_type {};
+struct __desugars_to<equal_to<_Tp>, std::equal_to<_Tp>> : true_type {};
```

Comment:
And you don't need to qualify `std::equal_to`, you can use just `equal_to` since it's not a function call, so there is no ADL to disable.